autoPyTorch/api/base_task.py

```diff
@@ -764,16 +851,22 @@ def _search(
 
         # ============> Run traditional ml
 
-        traditional_task_name = 'runTraditional'
-        self._stopwatch.start_task(traditional_task_name)
-        elapsed_time = self._stopwatch.wall_elapsed(self.dataset_name)
-        time_for_traditional = int(traditional_per_total_budget * max(0, (self._time_for_task - elapsed_time)))
-        if time_for_traditional <= 0:
-            if traditional_per_total_budget > 0:
```

Comment:
Is it okay to delete this ValueError?

Reply:
I have changed how the logic get's implemented, so it did not made sense to me to have it here. Now, if there is not enough time to fit a set of traditional pipelines, you will end up [here](https://github.com/automl/Auto-PyTorch/blob/0b641b0615c74e5d78d3b633caaee0b1d5bc509b/autoPyTorch/api/base_task.py#L668).

I can add an additional message if you think we need it, but fundamentally if the user provides a negative time for example, it means he wants to disable this feature.

Reply:
Ok, I just wanted to confirm